src/snowflake/cli/plugins/object/stage/commands.py

```diff
@@ -121,3 +123,27 @@ def stage_diff(
     """
     diff: DiffResult = stage_diff(Path(folder_name), stage_name)
     return ObjectResult(str(diff))
+
+
+@app.command("execute", requires_connection=True)
+def execute(
+    stage_path: str = typer.Argument(
+        None,
```

Comment:
```suggestion
        ...,
        show_default=False,
```
This way you'll get "missing argument" error, not "expected str (...), got NoneType" :)